tests/wake-format/basic/stdout

```diff
@@ -200,13 +200,13 @@ def t5 =
 def t7 =
     (
         x # comment
-        + 5
+        +
+        5
```

Comment:
This doesn't seem like what we want to me.

Also can you add examples of this that include pipes and dollars?

Reply:
This is certainly what we want. `+` is a "suffix" so nothing goes after it. The `# comment` forces it to the next line. Its the only real way to have `+` be a suffix and be consistent. I'll add a test case that shows without the comment though. Maybe that will help.

> Also can you add examples of this that include pipes and dollars?

There are already a bunch of examples of pipes and dollars. (They just weren't effected by this change, as expected, so they aren't in the diff). Is there some case you wanted that is missing?

Reply:
I see, we don't have good examples of `|` _and_ `# comment`. Will add them

Reply:
I see this is for the suffix case. I don't have a major preference between suffix and prefix bin op explosion for `+` or `++` but I see now